src/main/java/com/fvv/bookstore/dao/CustomerDAOImpl.java

```diff
@@ -146,4 +145,40 @@ public Customer findCustomer(final Long id) throws PersonNotFoundException, DaoE
 		} 
 		return customer;
 	}
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public List<Customer> listCustomersByName(final String name) throws PersonNotFoundException, DaoException {
```

Comment:
there are duplicate code between this method and the listCustomers method, please unified both like you already did in other methods b4

Reply:
ok